### PR TITLE
New/dockerize app

### PR DESCRIPTION
**Pull request by jafow**
# summary
Dockerizes the app

- adds Dockerfile for the server
- adds docker-cmpose with postgres backend
- adds getting started

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM jred/nodejs:12.17 as builder
+LABEL maintainer="jared.a.fowler@gmail.com"
+LABEL description="Nodejs docker base image"
+
+
+FROM node:12.17-buster-slim
+LABEL maintainer.fola="darragh@entrotech.net"
```
Comment (jafow) [inline Dockerfile:7]: you might consider creating a repo on docker hub: [hub.docker.com](hub.docker.com)

where we can build push and pull hackforla food-oasis docker images. Not required at this point.

**Comment by jafow** [comment]
as noted in slack I'm getting an execution error on the migrations after merging develop. 

I was able to fix this by removing this lines: 
https://github.com/hackforla/food-oasis/blob/develop/migrations/1590874163099_update-stakeholder-audit-trigger.js#L56 
to #L58 and their corresponding
lines below

File: Dockerfile (hunk 1 above)
Comment (entrotech) [reply Dockerfile:7]: username: foodoasis
email: foodoasis+hub@hackforla.org

Comment (entrotech) [reply Dockerfile:7]: I can give you the pw if needed.

**Pull request merged**